src/sagemaker_core/__init__.py

```diff
@@ -1,4 +0,0 @@
-from sagemaker_core.main.utils import enable_textual_rich_console_and_traceback
-
-
-enable_textual_rich_console_and_traceback()
```

Comment:
So by default we are disabling rich logging? Can we provide enable by default and prive an option to opt out? Our logging can provide directions to opt out.